jenkins/install-plugins.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+function print_usage() {
+  cat <<EOF
+Command
+  $0
+Arguments
+  --jenkins_url|-j          [Required]: Jenkins URL
+  --jenkins_user_name|-ju   [Required]: Jenkins user name
+  --jenkins_password|-jp              : Jenkins password. If not specified and the user name is "admin", the initialAdminPassword will be used
+  [..]                                : Plugins to install
```

Comment:
Can we ask for a comma separated list rather than extra parameters like this? I see that you modified the parameter handling and removed the error message, which feels like it could lead to bugs down the road